src/features/collaborator-avatars/CollaboratorListItem.js

```diff
@@ -38,7 +39,7 @@ const CollaboratorListItem = (props: Props) => {
                 <Link
                     href={profileURL || `/profile/${userID}`}
                     rel="noopener"
-                    target="_blank"
+                    target={isInsideIframe() ? '_self' : '_blank'}
```

Comment:
what's the current behavior if it's inside an iframe vs not in an iframe? is this considered a bug?

would there be a use case to use the current behavior? i'm wondering how it'll impact current usages since it may be expected now. e.g. if a customer is using ContentSharing inside of their own iframe

what if we pass `target` as a prop in `usernameProps` which is in `trackingProps`?